SKILL.tsv

```diff
@@ -1982,7 +1982,7 @@ SKILL_20150317_001981	$Adds additional damage to frozen enemies when attacking w
 SKILL_20150317_001982	$Rupture Blow
 SKILL_20150317_001983	Has a chance of inflicting damage with an explosion on nearby enemies when attacking with Hit-attacks. (5% chance of activating per level)
 SKILL_20150317_001984	$Rupture
-SKILL_20150317_001985	$Increases damage when using a Blunt weapon on chain armor-type enemies.{nl}(Increases by 20% per level)
+SKILL_20150317_001985	$Increases damage when using a Bludgeon weapon on Plate armor-type enemies.{nl}(Increases by 20% per level)
```

Comment:
This is talking about the weapon type, I don't know if its called [Blunt] or [Blunt Instrument].